### DISCUSSION TITLE
-gpt icon not there in dbeaver 23.1.5

### DISCUSSION THREAD

**Discussion by jgrover21** [dbeaver/dbeaver#21095] [Q&A] [answered]
-gpt icon not there in dbeaver 23.1.5

**Comment by E1izabeth** [comment]
Please explain in more details what do you mean

**Comment by E1izabeth** [comment, marked as answer]
To show up in a panel, these settings should be checked
![image](https://github.com/dbeaver/dbeaver/assets/28875055/97eeb278-667b-4604-b9f4-1da8fa5fb952)
![image](https://github.com/dbeaver/dbeaver/assets/28875055/34abf965-6209-4052-8bbd-83796118eb96)

**Reply by jgrover21** [reply]
I do not have that option available to select.
![image](https://github.com/dbeaver/dbeaver/assets/125315744/713b187a-59fa-435d-b286-9dc873a805e5)

**Reply by E1izabeth** [reply]
It means that the corresponding plugin is not installed.

**Comment by jgrover21** [comment]
How do I install those?

Respectfully,

Jyoti Grover


From: Elizabeth ***@***.***>
Sent: Wednesday, September 6, 2023 1:53 PM
To: dbeaver/dbeaver ***@***.***>
Cc: Grover, Jyoti ***@***.***>; Author ***@***.***>
Subject: <EXT> Re: [dbeaver/dbeaver] -gpt icon not there in dbeaver 23.1.5 (Discussion #21095)


It means that the corresponding plugin is not installed.

—
Reply to this email directly, view it on GitHub<https://github.com/orgs/dbeaver/discussions/21095#discussioncomment-6928763>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A54CVICJWEOCUGGWOBC63ITXZC2ABANCNFSM6AAAAAA4H64HTY>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

________________________________

CONFIDENTIALITY NOTICE: This email message and any attachments are confidential and are meant to be read by the intended recipient only. If you are not the intended recipient, please notify sender immediately and destroy all copies of this message and any attachments without reading or disclosing their contents. Thank you

**Reply by E1izabeth** [reply]
https://dbeaver.com/docs/dbeaver/AI-Smart-Assistance/#install-gpt-extension
